go/vt/srvtopo/watch_srvkeyspace.go

```diff
@@ -40,7 +40,7 @@ func (k *srvKeyspaceKey) String() string {
 func NewSrvKeyspaceWatcher(ctx context.Context, topoServer *topo.Server, counts *stats.CountersWithSingleLabel, cacheRefresh, cacheTTL time.Duration) *SrvKeyspaceWatcher {
 	watch := func(entry *watchEntry) {
 		key := entry.key.(*srvKeyspaceKey)
-		requestCtx, requestCancel := context.WithCancel(context.Background())
+		requestCtx, requestCancel := context.WithCancel(ctx)
```

Comment:
This was incorrectly using `context.Background()`. I also audited the paths into this function and from things like the CLI invocation, we do pass in the background context correctly.